cordformation/src/test/resources/net/corda/plugins/DeploySingleNodeWithCordappWithDocker.gradle

```diff
@@ -0,0 +1,53 @@
+buildscript {
+    ext {
+        corda_group = 'net.corda'
+        corda_release_version = '4.0'
+        jolokia_version = '1.6.0'
+        postgres_driver_version = '42.2.10'
+    }
+}
+
+plugins {
+    id 'net.corda.plugins.cordformation'
+}
+
+repositories {
+    mavenCentral()
+    maven { url 'https://software.r3.com/artifactory/corda-releases' }
+}
+
+dependencies {
+    runtime "$corda_group:corda:$corda_release_version"
+    runtime "$corda_group:corda-node-api:$corda_release_version"
+    cordapp "$corda_group:corda-finance-contracts:$corda_release_version"
+    cordapp "$corda_group:corda-finance-workflows:$corda_release_version"
+}
+
+task deployNodes(type: net.corda.plugins.Cordform) {
+    node {
+        projectCordapp {
+            deploy false
+        }
+        name 'O=Notary Service,L=Zurich,C=CH'
+        notary = [validating : true]
+        p2pPort 10002
+        rpcPort 10003
+        rpcSettings {
+            adminAddress "localhost:10004"
+        }
+        cordapps = ["$corda_group:corda-finance-contracts:$corda_release_version",
+                    "$corda_group:corda-finance-workflows:$corda_release_version"]
+    }
+}
+
+
+task prepareDockerNodes(type: net.corda.plugins.Dockerform) {
```

Comment:
Doesn't this version require you to specify what version of the postgres driver you want to get the databases? I'd have thought there'd be some changes to this task (and the ones below).

Reply:
Yes you need to add the postgres dependency version in the build.gradle file. Also you can specify the dockerImage in the same file. I will update the design document to reflect these changes